_data/projects/rubiks-cube.yml

```
@@ -0,0 +1,12 @@
+name: Rubik's Cube
+desc: A Rubik's Cube Visualizer with face turns, cube rotations, scramble generator and flat view of cube.
+site: https://github.com/vasu-gondaliya/rubiks-cube
+tags:
+  - html
+  - css
+  - javascript
+  - responsive
+  - website
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/vasu-gondaliya/rubiks-cube/labels/up%20for%20grabs
```

Looks like this label is out of sync:

```suggestion
  name: up-for-grabs
  link: https://github.com/vasu-gondaliya/rubiks-cube/labels/up-for-grabs
```